### PR TITLE
Multi bracket

### PR DESCRIPTION
**Pull request by rnmessick**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import java.util.ArrayList;
+
+public class MultiBracketValidation {
+    ArrayList<String> brackets = new ArrayList<>();
+
+    public  static  boolean multiBracketValidation(String input) {
+        if(input.length() == 0) {
+            throw new IllegalArgumentException("String length should be longer than 0");
+        }
+        if((input.length() % 2) != 0) {
```
Comment (evanslaton) [inline code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java]: The input length doesn't have to be even in order for this method to need to return true. If the input is `{odd}` the method should still return true
